frontend/src/libs/components/chat-sidebar/chat-element/chat-element.tsx

```diff
@@ -0,0 +1,47 @@
+import { PlainSvgIcon } from '#libs/components/components.js';
+import { IconNameToIcon } from '#libs/enums/enums.js';
+import { getValidClassNames } from '#libs/helpers/helpers.js';
+import { useCallback } from '#libs/hooks/hooks.js';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+  chat: Chat;
+};
+
+type Chat = {
+  id: number;
+  name: string;
+};
+
+// mocked data
+const selectedChat = {
+  id: 1,
+};
+
+const ChatElement: React.FC<Properties> = ({ chat }) => {
```

Comment:
we should make it as common component that we will reuse through the app
1. rename it to Card (as on mobile) 
2. move to `libs/components/card/card.tsx`
3. it should accept title, image, onPress
4. don't add trash icon for now

Reply:
@Liza-Veis can i also accept id of element?

Reply:
You don't need id in this component, but you can pass isActive

Reply:
let's do without arrow and trash icon for now, we will add it later

Reply:
@Liza-Veis so type for this component will look smth like this:
{
title:string;
imageId:number;
onPress:()=>void 
}

Reply:
No, we will pass image url, not image id

Reply:
@Liza-Veis how i could get id of the card and change active card if i don`t have id in card component?

Reply:
card is a stateless simple component that shouldn't know anything about logic, we will detect if it's active and handle active item change at the top level where we iterate through array. Like this
```
{chats.map((chat) => (
  <Card  isActive={chat.id === selectedChatId} onPress={() => setSelectedChatId(id)} />
))}
```
p. s. don't use the code above, it's just an example

Reply:
done